packages/gatsby-plugin-page-creator/src/derive-path.ts

```diff
@@ -51,6 +51,7 @@ export function derivePath(
     const replaceSlashesValue = (nodeValue + ``).replace(/\//g, `(REPLACED)`)
     const slugifiedWithoutSlashesValue = slugify(replaceSlashesValue, {
```

Comment:
@blainekasten could we use [@sindresorhus/slugify](https://npm.im/@sindresorhus/slugify) instead? It does what we need without any settings.

https://repl.it/@WardPeeters/slugify-package
![image](https://user-images.githubusercontent.com/1120926/92264206-2b9bc280-eede-11ea-96f3-35b6a6674620.png)

Reply:
Yeah, this looks much better! Switching over